src/ui/pages/work/Workorder.js

```diff
@@ -136,6 +138,40 @@ class Workorder extends Entity {
         }
     }
 
+    updateScheduleProperty = (key, value) => {
```

Comment:
Although this implementation can be used for the scheduled start/end dates of a Work Order, we still need the same behavior for the 'Add Activity' dialog start/end dates.

I believe the logic to add the days difference could be extracted in order to be reused.

Reply:
Separated the logic to a utils file, it's now used in both places